src/main/io/serial_4way.c

```diff
@@ -30,6 +30,10 @@
 
 #ifdef  USE_SERIAL_4WAY_BLHELI_INTERFACE
 
+#ifndef USE_PWM_OUTPUT
```

Comment:
Good find. But I think this should rather be solved by undefining `USE_SERIAL_4WAY_BLHELI_INTERFACE` in `common_post.h` if `USE_PWM_OUTPUT` is not defined, to bring it in line with how other features are handled.